src/s3.ts

```diff
@@ -7,31 +7,60 @@ const client = new s3({
 })
 
 export async function uploadVulnerability(rowJson: string): Promise<void> {
-  if (!process.env.BUCKET_NAME) {
+  if (!process.env.LOGS_BUCKET_NAME) {
     throw new Error('No bucket name.')
   }
-  const bucketName: string = process.env.BUCKET_NAME
+  const bucketName: string = process.env.LOGS_BUCKET_NAME
 
-  // Convert to JSON Lines
-  const json: string = JSON.stringify(JSON.parse(rowJson))
+  const json: string = convertToJsonLines(rowJson)
   core.debug(`JSON data: ${json}`)
 
   const param: s3.Types.PutObjectRequest = {
     Bucket: bucketName,
     Key: generateObjectKey('trivy', 'json'),
-    Body: `${json}\n`,
+    Body: json,
     ContentType: 'application/json',
     ACL: 'bucket-owner-full-control'
   }
 
+  s3PutObject(param)
+}
+
+export async function uploadBuildTime(
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  startTime: Date,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  endTime: Date
+): Promise<void> {
+  if (!process.env.METRICS_BUCKET_NAME) {
+    throw new Error('No bucket name.')
+  }
+  const bucketName: string = process.env.METRICS_BUCKET_NAME
+
+  const rowJson = '{}' // ToDo
+  const json: string = convertToJsonLines(rowJson)
+  core.debug(`JSON data: ${json}`)
+
+  const param: s3.Types.PutObjectRequest = {
+    Bucket: bucketName,
+    Key: generateObjectKey('build', 'json'),
```

Comment:
hive 形式にする必要があるかもです！

```suggestion
    Key: generateObjectKey('build/dt=', 'json'),
```

ref. https://github.com/C-FO/aws-infra/blob/master/alert-webhook/lambda/source/alert_webhook.py#L23

Reply:
全ファイルを取ってきても大した容量じゃないのでパーティション分割しないつもりでしたが、将来的にできるようにしておいた方がいいですね！
（パーティション分割は明示的に SQL を打つ必要があり、パスによって自動的に行われるわけではないです）

Reply:
Commit 47ef9cf で修正しました！

念のため修正後のコードで問題なく動くことは確認しました:ok_hand:
https://github.com/C-FO/build-image/runs/787116578?check_suite_focus=true